exercises/numbers/isNegative/isNegative.js

```diff
@@ -12,9 +12,16 @@ if (require.main === module) {
   console.log('Running sanity checks for isNegative:');
 
   console.log(isNegative(0) === false);
-  console.log(isNegative(1) === false);
-  console.log(isNegative(-1) === true);
-  console.log(isNegative(-100.201) === true);
+  // console.log(isNegative(1) === false);
+  // console.log(isNegative(-1) === true);
+  // console.log(isNegative(-100.201) === true);
+  return false; // not sure why this 'return' is appearing purple but others appear blue.
+} else if {
+  console.log(isNegative(num > 0) === false);
```

Comment:
Same here, calling `isNegative` from inside itself will cause it to run forever.

But also in this case, we _first_ call something like `isNegative(0)`.

Here's what `isNegative` "should" look like:

```js
function isNegative(num) {
  return num < 0;
}
```

One way to approach this is to look at known-working examples and trying to see how what you're trying to do is different from the known-working example. In this case, `isZero` is completed for you and serves as a representative "correct" solution.

I say "correct" because there are many ways to write the same function.

For example, you could also write `isNegative` like this (which might make more sense to you):

```js
function isNegative(num) {
  if (num < 0) {
    return true;
  } else {
    return false;
  }
}
```